app/assets/stylesheets/govuk_publishing_components/components/govspeak/_attachment.scss

```diff
@@ -19,42 +19,42 @@
 
   .attachment {
     position: relative;
-    margin: $gutter 0;
-    padding: $gutter-half 0 0 ($thumbnail-width + $gutter);
-    @extend %contain-floats;
+    margin: govuk-spacing(6) 0;
+    padding: govuk-spacing(3) 0 0 ($thumbnail-width + govuk-spacing(6));
+    @include govuk-clearfix;
 
     &:first-child {
       margin-top: 0;
       padding-top: 0;
     }
 
     .attachment-thumb {
-      $thumb-border-width: $gutter-one-third / 2;
+      $thumb-border-width: govuk-spacing(2) / 2;
```

Comment:
```suggestion
      $thumb-border-width: govuk-spacing(1);
```

Reply:
though I'm not convinced the spacing scale value should be used for width in the first place

Reply:
Good point @kr8n3r, we should use `$govuk-border-width` instead

Reply:
Replaced `$thumb-border-width` with `$govuk-border-width` so this definition is not needed anymore.